[MRG] Teaser for video-ident (#336)
BlockSigner/cove: site/content/english/_index.md

[//]: # (--------------------------------------------------------------------------------------------------------------)

{{< content text=center top=4 width=narrow >}}
{{% markdown %}}
Online identification via video

> or in a few days the whole organization

Reviewer: ```suggestion
or in a few days for your whole organization
```
What do you think?

===============
{{% /markdown %}}
{{< /content >}}

{{< content top=4 bottom=24 width=narrow display=flex >}}
{{< step >}}
{{< picture "video-ident" 250 "A magnifying glass enlarges the image of a man in a passport." >}}
{{% markdown %}}

> **Online identifikation**

Reviewer: ```suggestion
**Online identification**
```

> Enable the QES by getting yourself identified online via video-call or

Reviewer 2: ```suggestion
Enable the qualified electronic signature (QES) from home by getting yourself identified via video-call.
```

by personal contact with the ID app by Swisscom.
{{% /markdown %}}
<br>
{{< button
  "Learn more"
  "https://www.videoident.me/"
  "_blank"
  "outline">}}
{{< /step >}}

{{< step >}}
{{< picture "aes-for-biz-home-id" 280 "A big arrow, clicking on an organization, a high-rise building." >}}
{{% markdown %}}

> **Organisation-wide implementation with just a few clicks**

Reviewer 2: ```suggestion
**Organisation-wide implementation**
```

> Enable your entire staff to sign with the advanced electronic signature (AES) within just a few days.

Reviewer 2: ```suggestion
Enable the advanced electronic signature (AES) for your entire staff within just a few days.
```
